README.md

```diff
@@ -9,23 +9,25 @@ Hopefully your walk will be much easier than mine!
 
 ## Table Of Contents
 
-- [DAY 1](/day_01.md)
-  - [Introduction to JavaScript](/day_01.md#introduction-to-javascript)
-    - What is JS anyway?
-    - [What's ECMAScript?](/day_01.md#but-what-is-ecma)
-    - [Language features quick overview](/day_01.md#languages-features-quick-overview)
-    - [Myths busted](/day_01.md#myths-busted)
-  - [ES5 vs ES6](/day_01.md#es5-vs-es6)
-    - Relevant differences
-    - Relevant improvements
-    - [Why are we using ES6 in this course?](/day_01.md#why-are-we-using-es6-in-this-course)
-    - [After ES6?](/day_01.md#after-es6)
-  - [Initial workspace Setup](/day_01.md#initial-workspace-setup)
-    - Basic Runtime ( Node )
-    - IDE ( Visual Studio Code )
-    - [Run Code extension](/day_01.md#initial-workspace-setup)
-    - Introduction to NPM
-- [DAY 2](/day_02.md)
+
```

Comment:
Most of the corrections for README.md are related to markdownlint. (My fault I need to add a document with all collaboration rules 😢  ). Please add it to VSCode an it'll mark the required modifications, a configuration file is placed at the root level of the repo so that the exceptions are captured.

The rest of the corrections are related to "style" for this document. I'll list them here.

- Replace the H4 with H3 (e.g ##### [DAY 1](/day_01.md) with ### [DAY 1](/day_01.md)). Semantic meaning is preferred over look&feel
- I'm not a big fan of rulers especially on the TOC, let's try without and see how it looks